package/gluon-web-cellular/luasrc/lib/gluon/config-mode/model/admin/cellular.lua

```diff
@@ -33,6 +33,14 @@ local password = s:option(Value, "password", translate("Password"))
 password:depends(enabled, true)
 password.default = uci:get('gluon', 'cellular', 'password')
 
+local auth = s:option(ListValue, "auth", translate("auth"))
+auth:depends(enabled, true)
+auth:value("none", translate("none"))
+auth:value("pap", translate("pap"))
+auth:value("chap", translate("chap"))
+auth:value("both", translate("both"))
```

Comment:
User-facing capitalization is wrong (Also in translation files)

Reply:
sorry, what exactly do you mean?
Should it be e.g. "None", "Pap", "Chap", "Both"? 
if so i have changed it to that

Reply:
First argument to `value` is the value represented in UCI, this should be lowercase. The second argument given in `translate` is user-facing and should be capitalized as you did now.

Reply:
thanks, i think the following wording would be better, or?
`None`, `PAP`, `CHAP`, `Both`

Reply:
I prefer PAP and CHAP a lot more than Pap and Chap :)